sys/include/evtimer.h

```diff
@@ -56,7 +58,7 @@ extern "C" {
  */
 typedef struct evtimer_event {
     struct evtimer_event *next; /**< the next event in the queue */
-    uint32_t offset;            /**< offset in milliseconds from previous event */
+    uint32_t offset_ms;            /**< offset in milliseconds from previous event */
```

Comment:
Ok... now I'm not sure if the rename (which due to this line is also an API change) is really necessary. The doc (see also [line 25](https://github.com/RIOT-OS/RIOT/pull/8923/files#diff-86e356dc5223dce7b4926e951244cfc0R25)) already states in several places, that this timer is only using milliseconds, so a user using this timer *should* already be in clear that this PR uses only milliseconds.